tests/Maker/MakeTwigComponentTest.php

```diff
@@ -37,6 +37,28 @@ public function getTestDetails(): \Generator
             }),
         ];
 
+        yield 'it_generates_twig_component_in_non_default_namespace' => [$this->createMakerTest()
+            ->addExtraDependencies('symfony/ux-twig-component', 'symfony/twig-bundle')
+            ->run(function (MakerTestRunner $runner) {
+                $output = $runner->runMaker(['Alert']);
+
+                $this->assertStringContainsString('src/Site/Twig/Components/Alert.php', $output);
+                $this->assertStringContainsString('templates/components/Alert.html.twig', $output);
+                $this->assertStringContainsString('To render the component, use <twig:Alert />.', $output);
+
+                $runner->copy(
+                    'make-twig-component/tests/custom_twig_component.yaml',
```

Comment:
```suggestion
                    'make-twig-component/custom_twig_component.yaml',
```

That should do the trick. Looks like the yaml file is in `tests/fixtures/make-twig-component/custom_twig_component.yaml`